### PR TITLE
[WIP] script changes to check if edgecore is running then Kill it

### PR DESCRIPTION
**Pull request by srivatsav123**
These script changes will kill the edgecore when Edgecore is Running

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: edge/test/integration/scripts/execute.sh [hunk 1]
```diff
@@ -29,14 +29,28 @@ go build cmd/edge_core.go
 #dynamically append testManager Module before starting integration test.
 sed -i 's/dbTest/dbTest, testManager/g' conf/modules.yaml
 #restart edge_core after appending testManager Module.
+
+if pgrep edge_core >/dev/null ; then
+echo "Old edge core is running"
```
Comment (yongwww) [inline edge/test/integration/scripts/execute.sh]: better to have indentation identical for the whole file

File: edge/test/integration/scripts/execute.sh [hunk 2]
```diff
@@ -29,14 +29,28 @@ go build cmd/edge_core.go
...
+while true; do
 sudo pkill edge_core
+if [ $? == 0 ]; then
+echo "Edgecore got killed"
+break
+fi
+done
+else
+echo "No Old edge core is running"
```
Comment (yongwww) [inline edge/test/integration/scripts/execute.sh]: typo: old

File: edge/test/integration/scripts/execute.sh [hunk 3]
```diff
@@ -29,14 +29,28 @@ go build cmd/edge_core.go
...
+
 #kill the edge_core process if it exists, wait 2s delay before start the edge_core process.
 sleep 2s
 nohup ./edge_core > edge_core.log 2>&1 &
 sleep 15s
 if pgrep edge_core >/dev/null
 then
-    echo "edge_core process is Running"
+    pid=$(lsof -i tcp:10255 | awk '{print $2}' | awk 'FNR == 2 {print}')
+    echo "edge_core process has Started pid is $pid"
```
Comment (yongwww) [inline edge/test/integration/scripts/execute.sh]: typo

File: edge/test/integration/scripts/execute.sh [hunk 4]
```diff
@@ -54,7 +68,20 @@ bash -x ${PWD}/scripts/fast_test $1
 #Reset env
 sed -i 's/dbTest, testManager/dbTest/g' conf/modules.yaml
 #stop the edge_core after the test completion
+
+if pgrep edge_core >/dev/null ; then
```
Comment (yongwww) [inline edge/test/integration/scripts/execute.sh]: The section here was defined before, reuse it as a function?

File: edge/test/integration/device/device_test.go [hunk 5]
```diff
@@ -116,57 +118,94 @@ var _ = Describe("Event Bus Testing", func() {
 		})
 
 		It("TC_TEST_EBUS_1: Sending data to Cloud", func() {
-
+			cmd := exec.Command("sh", "-c", "lsof -i tcp:10255 | awk '{print $2}' | awk 'FNR == 2 {print}'")
```
Comment (yongwww) [inline edge/test/integration/device/device_test.go]: just for curiosity, 10255 is a default port?

**Comment by sids-b** [comment]
@yongwww ,we were testing flaky integration tests here. Please ignore this PR

**Pull request closed without merging**